server/liveblog/themes/themes.py

```diff
@@ -600,24 +601,26 @@ def _get_publishing_step(self, theme):
                 return STEPS.get(key)
         return STEPS.get("default")
 
-    def _schedule_items(self, items, step, is_blog):
-        """Schedules publishing for given items."""
+    def _schedule_items(self, items, step, is_blog, batch_size=10, delay=1):
+        """Schedules publishing for given items in manageable batches."""
```

Comment:
can you add more context about why we use batches here and what's the logic behind it?